### PR TITLE
DF-150: Added titles on every table and fixed the alignment of the table

### PR DESCRIPTION
**Pull request by RITIKA-SHARMAA**

### CODE REVIEW

**Comment by Akhil-chaudhary** [comment]
Add screenshots or description to elaborate the task

**Comment by RITIKA-SHARMAA** [comment]
Screenshot 2024-11-03 at 12 53 13 AM


Screenshot 2024-11-03 at 12 53 29 AM

Screenshot 2024-11-03 at 12 57 05 AM


Screenshot 2024-11-03 at 12 57 40 AM

added titles on every tables and fixed the alignment of the table

File: src/components/Admin/adminTable.tsx [hunk 1]
```diff
@@ -91,10 +91,14 @@ const AdminTable: React.FC = () => {
...
-        <div className="flex items-center justify-between mb-4">
-          <div className="flex space-x-4">
-            <AddAdmin onAddAdmin={handleAddAdmin} />
+            <div className="mb-8 mt-4 ">
+              <div className="flex items-center justify-between mb-4 ">
+                <div className="flex-grow">
+                  <h2 className="text-xl font-semibold">Admin Table</h2>
+                </div>
+                <div>
+                  {" "}
```
Comment (Divesh-singh) [inline src/components/Admin/adminTable.tsx:100]: Don't use {" "} to add spacing to title, use css for this kind of things
update all instances in PR where this is used

Comment (Divesh-singh) [reply src/components/Admin/adminTable.tsx]: fix all the instances of this  {" "} approach

**Pull request merged**